test/suites/common-tanssi/pallet-data-preservers/test_pallet_data_preservers.ts

```diff
@@ -37,12 +38,14 @@ describeSuite({
                 const storedProfile = await polkadotJs.query.dataPreservers.profiles(profileId);
                 expect(storedProfile.toJSON()).to.be.deep.equal({
                     account: general_user_bob.address,
-                    deposit: 10_190_000_000_000,
+                    deposit: 10_200_000_000_000,
```

Comment:
Can we add start_assignemnt and force_start_assignment as well as stop_assignment tets here?